src/tests/Interop/IDynamicInterfaceCastable/Program.cs

```diff
@@ -334,6 +334,15 @@ int ITest.GetNumber()
         }
     }
 
+    public struct ValueTypeDynamicInterfaceCastable : IDynamicInterfaceCastable
+    {
+        public bool IsInterfaceImplemented(RuntimeTypeHandle interfaceType, bool throwIfNotImplemented)
+            => throw new Exception("ValueType implementations are ignored");
```

Comment:
```suggestion
            => throw new UnreachableException("ValueType implementations are ignored");
```

Use UnreachableException for slightly more clarity?

Reply:
Good idea. Done.